File: src/main/java/com/musinsa/subject/model/price/LowestPriceByCategoryResponse.java
package com.musinsa.subject.model.price;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@Builder
public class LowestPriceByCategoryResponse {

    public record Category(String categoryName, String brandName, String productPrice) {}

    private List<Category> categoryLowestPrices;
    private String totalPrice;

}

File: src/main/kotlin/com/musinsa/subject/model/price/LowestPriceByCategoryResponse.kt
package com.musinsa.subject.model.price

class LowestPriceByCategoryResponse(
    var categoryLowestPrices: List<Category>? = null,
    var totalPrice: String? = null
) {
    data class Category(
        val categoryName: String,
        val brandName: String,
        val productPrice: String
    )
}
